resources/multisite_drupal_standard.make

```diff
@@ -616,7 +616,7 @@ projects[video][patch][] = patches/video-security-883.patch
 ; See https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-7788
 ; Todo: revert back to the stable version when Views 7.x-3.14 is released.
 projects[views][download][branch] = 7.x-3.x
-projects[views][download][revision] = e5c8b65
```

Comment:
It should be fixed on a separate ticket. A JIRA number should be mentioned on the commit message.